res/values-de/strings.xml

```diff
@@ -284,4 +284,6 @@ es welche gibt) anzeigen lassen.</string>
   <string name="catalan">Katalanisch</string>
   <string name="polish">Polnisch</string>
   <string name="tamil">Tamil</string>
+  <string name="cancel">Abbrechen</string>
+  <string name="ok">Ok</string>
```

Comment:
Thanks, but there's no need adding them here, as this is overwritten when pulled from Transifex next time.